screens/index.js

```diff
@@ -3,4 +3,9 @@
  */
 export { default as Dashboard } from './Dashboard/Dashboard';
 export { default as DraftsPage } from './DraftsPage/DraftsPage';
-export { default as Settings } from './Settings/Settings';
\ No newline at end of file
+export { default as Settings } from './Settings/Settings';
+export { default as LoginScreen } from './LoginScreen/LoginScreen';
+export { default as RegisterScreen } from './RegisterScreen/RegisterScreen';
+export { default as MainScreen } from './MainScreen/MainScreen';
+
+export { default as LoginRegisterStyleCheck } from './LoginRegisterStyleCheck/LoginRegisterStyleCheck';
```

Comment:
The app runs fine without this import, when I run it with this import I get this error:
```
iOS Bundling failed 21473ms (node_modules/expo/AppEntry.js)
Unable to resolve "./LoginRegisterStyleCheck/LoginRegisterStyleCheck" from "screens/index.js"
```

Reply:
Yeah I left that there by mistake, should be removed apologies